interp/run.go

```diff
@@ -1070,7 +1070,7 @@ func callBin(n *node) {
 	rcvrOffset := 0
 	if recv := n.child[0].recv; recv != nil && !isInterface(recv.node.typ) {
 		numIn, numChild := funcType.NumIn(), len(child)
-		if variadic > 0 && numIn >= numChild || numIn > numChild {
+		if variadic > 0 && numIn > variadic || numIn > numChild {
```

Comment:
given that 
`numIn := funcType.NumIn()` and
`variadic := funcType.NumIn() - 1`
then `numIn > variadic` is always true if variadic != -1. So I think this is redundant, and it should just be:
`if variadic > 0 || blabla ...`
right?

Reply:
right.